dropwizard-e2e/src/main/java/com/example/app1/App1.java

```diff
@@ -8,13 +8,16 @@
 import io.dropwizard.setup.Bootstrap;
 import io.dropwizard.setup.Environment;
 import io.dropwizard.views.ViewBundle;
+import org.eclipse.jetty.io.EofException;
 import org.glassfish.jersey.spi.ExtendedExceptionMapper;
 
 import javax.ws.rs.WebApplicationException;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.ext.ExceptionMapper;
 
 public class App1 extends Application<Configuration> {
+    public boolean wasEofExceptionHit = false;
```

Comment:
Shouldn't the variable be volatile? It's written in a Jetty's app thread and read in a test executor thread.

Reply:
Ah yes, I have to keep reminding myself [about the correct usage](https://www.ibm.com/developerworks/java/library/j-jtp06197/). Since this is an example of a status flag, volatile should be used here. I have updated the PR